playbooks/appserver-init.yml

```diff
@@ -0,0 +1,13 @@
+---
+# Playbook to set up common services on Open edX appservers.  These services run before setting up
+# Open edX: The consul_connect role sets up the tunnels to Redis and Memcached, which need to be in
+# place before running the edx-configuration playbook.
+
+- name: Set up common services on Open edX appservers – initialization
+  hosts: app
+  become: true
+  roles:
+    - role: consul
+      when: consul_servers | length > 0
+    - role: consul-connect
+      when: consul_connect_instance_id
```

Comment:
In practice, IDs start at 1, so this simplify won't be zero.  In theory, it would be valid to have an ID of zero, for example by manually assigning this ID to an instance, so I can change this.

I can also never remember what the correct check here is.  The expression is evaluated via Jinja2, so this is not a Python expression.  The value defaults to `null` in YAML, which translates to `None` in Python, so we need to use the [built-in `none` check from Jinja2](http://jinja.pocoo.org/docs/2.10/templates/#none), so the expression becomes

    when: consul_connect_instance_id is not none

with a lower-case "n".  This part of Ansible certainly isn't the most ergonomic.